rest-api-spec/src/main/resources/rest-api-spec/test/tasks.list/10_basic.yaml

```diff
@@ -0,0 +1,6 @@
+---
+"tasks_list test":
+  - do:
+      tasks.list: {}
+
+  - is_true: nodes
```

Comment:
Hmm - it'd be nice to have more to check here but I understand there not being much now. We can come up with _something_ eventually.